src/components/LaunchDetailsPage/components/LaunchCardContent/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import { launchDetails } from '../../../../mocks/launchDetails';
+import './CardDetails.scss';
+
+export const CardDetails = () => {
+  const detailsCaptions: string[] = ['Area:', 'Status:', 'Date:'];
```

Comment:
create enum for captions and move to constants

Reply:
done